django/thunderstore/repository/migrations/0055_delete_namespaces_without_team.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.1.7 on 2025-01-07 15:07
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    Namespace = apps.get_model("repository", "Namespace")
+
+    Namespace.objects.filter(team__isnull=True).delete()
```

Comment:
This will break if the namespace has packages associated to it, so you should filter out namespaces with packages as well. I don't think the production db has any such instances but it is technically a bug so mentioning anyway.